test/test_blockviews.jl

```diff
@@ -45,6 +45,7 @@ bview(a, b) = Base.invoke(view, Tuple{AbstractArray,Any}, a, b)
         @test size(V) == (2, 5)
         V[1,1] = -1
         @test A[2,8] == -1
+        @test A[Block(4)] == A[Block(1),Block(2)]
```

Comment:
Can you add
```julia
@test_throws BlockBoundsError A[Block(5)]
```

Reply:
I think `Block(5)` is in bound. Maybe `Block(10)` instead?